addons/sys_io/fnc_ts3ChannelCheck.sqf

```diff
@@ -0,0 +1,25 @@
+/*
+ * Author: Tim Beswick
+ * Checks if main display is visible and sets server name, triggering ts3 channel move.
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [] call acre_sys_server_fnc_teamspeakChannelCheck
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+[{
+    params ["", "_idPFH"];
+
+    if (GVAR(serverStarted) && {!(isNull (findDisplay 46))}) exitWith {
```

Comment:
Ah, I see. In that case we should use the CBA Display EH:
```
class Extended_DisplayLoad_EventHandlers {
    class RscDisplayMission {
        ADDON = QUOTE(_this call COMPILE_FILE(XEH_missionDisplayLoad.sqf));
    };
};
```
And obviously move that code to the `XEH_missionDisplayLoad.sqf` file. No need for a PFH then.